sdk/storage/azure-storage-queue/src/test/java/com/azure/storage/queue/spock/QueueAPITests.groovy

```diff
@@ -0,0 +1,485 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.queue.spock
+
+
+import com.azure.storage.queue.models.AccessPolicy
+import com.azure.storage.queue.models.SignedIdentifier
+import com.azure.storage.queue.models.StorageErrorCode
+import com.azure.storage.queue.models.StorageErrorException
+import org.junit.Assert
+import org.junit.Ignore
+import reactor.test.StepVerifier
+import spock.lang.Unroll
+
+import java.time.Duration
+import java.time.LocalDateTime
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+
+import static org.junit.Assert.assertEquals
```

Comment:
Could we leverage Spock's `then` and `expect` steps which act as assertions.

Reply:
Will double check if we can remove the assertion.

Junit still need exist as @Rule annotation depends on it.